apps/evm/test/pages/swap.test.ts

```diff
@@ -184,13 +196,21 @@ async function swap(page: Page, inputCurrency: Type, outputCurrency: Type, amoun
   await expect(swapFromBalance).toBeEnabled()
   const swapFromBalanceBefore = await swapFromBalance.textContent()
   const swapToBalance = page.locator('[testdata-id=swap-to-balance-button]')
-  await expect(swapToBalance).toBeVisible()
+  await expect(swapToBalance).toBeVisible({ timeout: 120_000 })
```

Comment:
Can't see why arb timeout is needed here?

Reply:
It took _very_ long for me sometimes, wanted to make sure it wouldn't just time out